packages/flutter_tools/lib/src/build_system/targets/web.dart

```diff
@@ -470,6 +470,7 @@ const CORE = [
   ${coreBundle.map((String file) => '"$file"').join(',\n')}];
 // During install, the TEMP cache is populated with the application shell files.
 self.addEventListener("install", (event) => {
+  self.skipWaiting();
```

Comment:
`skipWaiting` doesn't seem to be supported in Safari: https://caniuse.com/?search=skipWaiting

Reply:
there is no mechanism to force update (automatically or via user interaction) without skip waiting.

Should we disable service workers on safari?

Reply:
This does work on safari desktop it seems, but it is not sufficient on iOS

Reply:
I'm guessing it's less of a concern on mobile since on mobile you typically don't have multiple tabs of the same app active at the same time?

Reply:
🤷‍♂️